### PR TITLE
Feature/parameterise celltype field

### PR DESCRIPTION
**Pull request by pinin4fjords**
This is the update to the workflows collection (comprising also https://github.com/ebi-gene-expression-group/scxa-bundle-workflow/pull/8), to be paired with https://github.com/ebi-gene-expression-group/scxa-control-workflow/pull/22.

### CODE REVIEW

File: w_droplet_clustering/scanpy_clustering_workflow_parameters.yaml [hunk 1]
```diff
@@ -75,7 +75,7 @@ find_markers:
         reference: rest
         use_raw: 'true'
 find_markers_cell_type:
-    groupby: inferred_cell_type
+    groupby: CELL_TYPE_FIELD
```
Comment (pcm32) [inline w_droplet_clustering/scanpy_clustering_workflow_parameters.yaml:78]: I'm guessing that if the scanpy module for finding marker genes doesn't find this field for groupby doesn't fail or the failure is handled at the level of data set collection filtering? Just wondering.

**Review by pcm32 — APPROVED** [review]
LGTM

File: w_droplet_clustering/scanpy_clustering_workflow_parameters.yaml (hunk 1 above)
Comment (pinin4fjords) [reply w_droplet_clustering/scanpy_clustering_workflow_parameters.yaml:78]: The Galaxy workflow step that does cell type marker detection is allowed to fail.

**Comment by pinin4fjords** [comment]
Thanks @pcm32

**Pull request merged**